ui/src/app/app.tsx

```diff
@@ -110,7 +98,17 @@ export class App extends React.Component<{}, {version?: Version; popupProps: Pop
 
     public componentDidMount() {
         this.popupManager.popupProps.subscribe(popupProps => this.setState({popupProps}));
-        services.info.getVersion().then(version => this.setState({version}));
+        services.info
```

Comment:
🔑  This is one of the key changes in this PR. This allows us to know what managed namespace we're running in almost immediately. We use this to create redirects that mean that child components no longer need to know about namespaces.

Reply:
Great change